src/_sass/footer.scss

```diff
@@ -23,4 +23,15 @@
     &:hover {
         filter: brightness(80%);
     }
+}
+
+@media (max-width: 500px) {
```

Comment:
just make sure to check if that is a good breakpoint.

Reply:
Yes! I checked it on chrome's responsive mode and it looked good to me